File: PJS5_POC_Cpp/Sources/Weapon/NormalWeapon/NormalWeapon.cpp
#include "NormalWeapon.h"

namespace WEAPON
{
	NORMAL_WEAPON::NORMAL_WEAPON(const int iId, const std::string& sName, const int iArmor, const int iLifePoint, const int iDamage, const int iPowerConsumption, const int iAccuracy, const int iMaxInaccuracy)
		: IWEAPON::IWEAPON(iId, sName, iArmor, iLifePoint, iDamage, iPowerConsumption, iAccuracy, iMaxInaccuracy, WEAPON_TYPE::NORMAL_WEAPON)
	{
	}
}

File: PJS5_CSharp/PJS5_CSharp/Sources/Weapon/NormalWeapon/NormalWeapon.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WEAPON;

namespace PJS5_CSharp.Sources.Weapon.NormalWeapon
{
    public class NORMAL_WEAPON : IWeapon
    {
        public NORMAL_WEAPON(int iId, string sName, int iArmor, int iLifePoint, int iDamage, int iPowerConsumption, int iAccuracy, int iMinAccuracy)
            : base(iId, sName, iArmor, iLifePoint, iDamage, iPowerConsumption, iAccuracy, iMinAccuracy, WEAPON_TYPE.NORMAL_WEAPON)
        {
        }
    }
}
